addons/locales.sh

```diff
@@ -5,11 +5,8 @@
 # shellcheck disable=2034,2059,1091
 true
 SCRIPT_NAME="Locales"
-
 # shellcheck source=lib.sh
-source /var/scripts/fetch_lib.sh &>/dev/null || . <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh) &>/dev/null
-
-###################################
```

Comment:
2. I don't like the border here (but I understand what you wanted to do, I only think it is not necessary)

Reply:
It marks where the header ends. Everything above that line is static. Easier to find.

Reply:
Hm, I think they are still very easy to find also without, because they sit at the top.
And what about those in other scripts: aren't they headers, too?
```
# Check for errors + debug code and abort if something isn't right
# 1 = ON
# 0 = OFF
DEBUG=0
debug_mode

# Must be root
root_check
```

Reply:
So IMO we move the border below `root_check` or just remove the border completely.

Reply:
Sure, let's move if to after `root_check`.

Reply:
done